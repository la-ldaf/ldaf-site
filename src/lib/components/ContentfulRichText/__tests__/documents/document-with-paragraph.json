{
  "nodeType": "document",
  "data": {},
  "content": [
    {
      "nodeType": "paragraph",
      "data": {},
      "content": [
        {
          "nodeType": "text",
          "value": "This text is ",
          "data": {},
          "marks": []
        },
        {
          "nodeType": "text",
          "value": "important",
          "data": {},
          "marks": [{ "type": "bold" }]
        }
      ]
    }
  ]
}
