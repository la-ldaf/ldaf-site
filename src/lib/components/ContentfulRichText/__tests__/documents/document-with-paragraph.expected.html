<p>This text is <strong>important</strong></p>
